dev-docs/DEV_ENVIRONMENT.md

```diff
@@ -78,27 +78,27 @@ password, respectively.
 
 ## Setup
 
-To be able to build and run A2 on your workstation, you need the
+To be able to build and run Automate on your workstation, you need the
 following prerequisites:
 
-1. Clone A2 repo locally
+1. Clone Automate repo locally
 
    From a terminal: run `git clone https://github.com/chef/automate.git`
 
-1. Install `direnv` and integrate it into your shell
+2. Install `direnv` and integrate it into your shell
 
-   a. First, install the package. On Homebrew: run `brew install direnv`, on Ubuntu: run `apt-get install direnv`
-   b. Next, [hook direnv into your shell](https://github.com/direnv/direnv#setup)
-   c. Finally, run `direnv allow` from the A2 source directory.
+   1. First, install the package. On Homebrew: run `brew install direnv`, on Ubuntu: run `apt-get install direnv`
+   2. Next, [hook direnv into your shell](https://github.com/direnv/direnv#setup)
+   3. Finally, run `direnv allow` from the Automate source directory.
 
-1. Install and configure Habitat
+3. Install and configure Habitat
 
    From a terminal:
 
-   a. To install version `0.59.0` of Habitat:
-  `curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash -s -- -v 0.59.0`
-
-   b. To configure the Habitat cli run and follow the instructions:
+   1. Determine the supported version of Habitat. It is 0.77.0 at the time of writing this, but the current version can be found in [studio-common's `.hab-version` file](https://github.com/chef/studio-common/blob/master/.hab-version).
+   2. To install version `0.77.0` of Habitat:
+  `curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash -s -- -v 0.77.0` or refer to the [Habitat installation guide](https://www.habitat.sh/docs/install-habitat/).
+   3. To configure the Habitat cli run and follow the instructions:
```

Comment:
If someone followed these instructions exactly they would install an outdated version of Habitat and would see a big scary warning when starting the studio that they are using an unsupported version of Habitat.

Reply:
👍 As a follow up, my vote is that we remove the check.  We should be encouraging people to aggressively upgrade hab in the studio so we can find errors.